net/unbound/files/unbound-odhcpd.defaults

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+[ "$(uci get dhcp.odhcpd.maindhcp)" = "1" ] || exit 0
+[ "$(uci get unbound.ub_main.dhcp_link)" = "none" ] || exit 0
+
+[ -x /usr/sbin/unbound-control ] && uci set unbound.ub_main.unbound_control='1'
+
+uci batch <<EOF
+set unbound.ub_main.add_local_fqdn='1'
+set unbound.ub_main.add_wan_fqdn='1'
+set unbound.ub_main.dhcp_link='odhcpd'
+set unbound.ub_main.dhcp4_slaac6='1'
+set unbound.ub_main.domain='lan'
```

Comment:
You want to be sure to set this to the same in `/ect/config/dhcp` or check that they match or check that this is default "lan" while DHCP UCI is not default "lan". Maybe check before doing any of this. It would be a good defense against unintended consequences of overriding defaults.

Reply:
Good point.